hw/ip/rstmgr/data/rstmgr.hjson

```diff
@@ -219,7 +219,7 @@
 
     { multireg: {
         cname: "RSTMGR_SW_RST",
-        name:  "SW_RST_REGEN",
+        name:  "SW_RST_REGWEN",
```

Comment:
that's weird..i thought i already merged a PR to do this...

Reply:
Apparently you just did the .tpl / autogen files.

Reply:
And this only makes the names consistent with the top_earlgrey names, so no real name change.

Reply:
o sounds good.